### PR TITLE
#166790041 end to end tests setup

### PR DESCRIPTION
**Pull request by pngabo**
#### What does this PR do?
 - Sets up end to end tests
#### Description of Tasks to be completed?
 - setting up end to end tests
#### How should this be manually tested?
 - clone the repository with https://github.com/andela/ah-jawans-frontend.git
 - npm install
 - npm run test
#### Any background context you want to provide?
 - End-to-end testing is a technique used to test whether the flow of an application right from start to finish is behaving as expected

#### What are the relevant pivotal tracker stories?
[#166790041](https://www.pivotaltracker.com/story/show/166790043)

Screenshots (if appropriate)
 
image

### CODE REVIEW

File: __tests__/components/index.test.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+it('Should pass', () => {
+  expect(5 * 5).toEqual(25);
+});
```
Comment (Joe-Joseph) [inline __tests__/components/index.test.js]: Why did you name test folder like that, with underscores

File: __tests__/setup/setupEnzyme.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16.3';
+
+Enzyme.configure({ adapter: new Adapter() });
```
Comment (Joe-Joseph) [inline __tests__/setup/setupEnzyme.js]: Please add a new line at the end of every file

**Comment by mnzube** [comment]
Hey Patrick, LGTM except that I think you forgot to make the pivotal tracker story number a link. Also on 'what this PR does?' question I think you should have 'Sets up end to end tests ' instead of setup.

**Review by shaluchandwani — APPROVED** [review]
Good work

File: __tests__/components/index.test.js (hunk 1 above)
Comment (pngabo) [reply __tests__/components/index.test.js]: Underscores and double underscores are commonly used to denote things that are outside of the regular code.

**Review by Joe-Joseph — APPROVED** [review]

**Review by mnzube — APPROVED** [review]
Nice

**Review by kagabof — APPROVED** [review]
LGTM

**Pull request merged**